app/measure/measure.html

```diff
@@ -1,8 +1,11 @@
 <!-- Intro -->
 <section id="intro" class="wrapper style1 fullscreen">
   <div class="featureWarning" ng-show="isSafari">
-    <p translate>We're sorry, the Safari browser is not compatible with the NDT
-    test. Please use a different browser such as Chrome or Firefox instead.</p>
+    <p translate>We're sorry, but M-Lab has identified bugs with <a
+    href="https://webkit.org/">Safari's websockets</a> implementation that
+    results in innacurate results when using the browser to run the NDT test
+    (both ndt7 and ndt5 versions). Please use a different browser such as Chrome
+    or Firefox instead. For more information, please see this <a href="https://github.com/m-lab/ndt7-js/issues/19">issue</a>.</p>
```

Comment:
This issue link is good, but I believe there are other issues not identified here.  It looks like the onebox test succeeds on my laptop in Safari, but the download throughput is poor.   Don't have any recommended change - just want to have everyone on the same page.

Does anyone know enough about the onebox client to guess what might be going on there?

Reply:
The download test is slow I believe due to a lot of garbage collection happening while receiving data, I don't think it can be mitigated. Nothing like that happens on other browser engines.

The upload test bug, which causes the client to crash after reporting an insanely high upload rate for the first 1-2 measurements (0.5s), can be mitigated by disabling message size scaling and reducing `desiredBuffer`, which is what the OneBox client does.

Reply:
For a general audience, I think this link is more than enough. For folks who want to learn more they have the link to our github issue below.